presto-main/src/test/java/com/facebook/presto/operator/scalar/FunctionAssertions.java

```diff
@@ -290,6 +290,11 @@ public void executeProjectionWithFullEngine(String projection)
         MaterializedResult result = runner.execute("SELECT " + projection);
     }
 
+    protected Object selectStringValue(String projection, Type expectedType)
```

Comment:
Appears unused?

Reply:
It is used in TestQuantileDigestFunctions

Reply:
Oh I see.  This doesn't necessarily return a String value.  I recommend changing the signature to `protected <T> T selectSingleValue(String projection, Type expectedType, Class<T> clazz)`, and asserting that the returned value is actually an instance of `clazz`, before finally casting to `T` and returning.